### ISSUE TITLE
After long press of a button the same button did not work the next time you press it

### ISSUE THREAD

**Issue by Kat-CeDe** [unfoldedcircle/integration-androidtv#48]
### Description

I have back set to back and long press back to switch back to my previous IPTV channel. If I long press back and then again back this command is not send.

### How to Reproduce

1. Define a command to a button
2. Define a command to the same button but as long press
3. long press define button
4. command is executed
5. short press button
6. button is ignored

### Expected behavior

6. command is executed even after long press as previous button

### Integration version

0.5.1

**Comment by zehnm** [comment]
I can't reproduce this with the following test case:
1. Create activity
2. Add AndroidTV media-player entity to activity
3. Map `cursor up` to back key, short-press 
4. Map `cursor down` to back key, long-press
5. Add activity to UI
6. Start activity
7. Long-press back key: selection moves down on ATV
8. Short-press back key: selection moves up on ATV

Please check the integration log file if the integration receives both commands.

**Comment by Kat-CeDe** [comment]
You are right I tested with cursor left and right there it also works for me.

My setup was a bit different:
short back = back
long back = macro with cursor down, delay 500ms, cursor enter

With sequence I can toggle between 2 IPTV channel. This still has the effect that it does not work twice in a row. The 3rd press on the same button is then working again.
